pypdf/generic/_data_structures.py

```diff
@@ -1443,9 +1454,13 @@ def read_object(
         else:
             return NumberObject.read_from_stream(stream)
     else:
+        pos = stream.tell()
         stream.seek(-20, 1)
+        txt = stream.read(80)
```

Comment:
This name is misleading - this isn't really text, but rather bytes, if I am not mistaken.

Reply:
will change to stream_extract